apps/web/pages/api/intercom-hash.ts

```diff
@@ -0,0 +1,28 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import crypto from "node:crypto";
+import { z } from "zod";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { query, method } = req;
+
+  const id = z.string().safeParse(query?.id);
+
+  switch (method) {
+    case "GET":
+      if (id.success) {
+        // eslint-disable-next-line turbo/no-undeclared-env-vars
+        const hmac = crypto.createHmac("sha256", process.env.INTERCOM_SECRET ?? "");
```

Comment:
It looks like a new env variable. It should be added in .env.example with comment on when/why it should be set.